charts/enterprise-metrics/templates/_helpers.tpl

```diff
@@ -59,3 +59,10 @@ Create the app name of enterprise-metrics clients. Defaults to the same logic as
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Calculate the config from structured and unstructred text input
```

Comment:
```suggestion
Calculate the config from structured and unstructured text input
```

Reply:
we've removed structured anyway, rewriting this